### PR TITLE
docs: update devops flight-manuals

### PR DESCRIPTION
**Pull request by raisedadead**
Adds/updates instructions for operators to provision chat, Azure pipelines and other misc. updates

### CODE REVIEW

File: docs/devops.md [hunk 1]
```diff
@@ -740,3 +642,295 @@ pm2 reload all --update-env && pm2 logs
 
 > [!NOTE] We are handling rolling updates to code, logic, via pipelines. You
 > should not need to run these commands. These are here for documentation.
+
+## Work on Chat Servers
+
+Our chat servers are available with a HA configuration [recommend in Rocket.Chat docs](https://docs.rocket.chat/installation/docker-containers/high-availability-install). The `docker-compose` file for this is [available here](https://github.com/freeCodeCamp/chat-config).
```
Comment (nhcarrigan) [inline docs/devops.md:648]:
```suggestion
Our chat servers are available with a HA configuration [recommended in Rocket.Chat docs](https://docs.rocket.chat/installation/docker-containers/high-availability-install). The `docker-compose` file for this is [available here](https://github.com/freeCodeCamp/chat-config).
```

File: docs/devops.md [hunk 2]
```diff
@@ -740,3 +642,295 @@ pm2 reload all --update-env && pm2 logs
...
+     - all stopped containers
+     - all networks not used by at least one container
+     - all volumes not used by at least one container
+     - all dangling images
+     - all dangling build cache
+
+   Are you sure you want to continue? [y/N] y
+   ```
+
+   Select yes (y), to remove everything that is not in use.
```
Comment (nhcarrigan) [inline docs/devops.md:824]:
```suggestion
   Select yes (y) to remove everything that is not in use.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Couple of tiny tweaks

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**